modules/stringtie/stringtie/main.nf

```diff
@@ -48,4 +48,18 @@ process STRINGTIE_STRINGTIE {
         stringtie: \$(stringtie --version 2>&1)
     END_VERSIONS
     """
+
+    stub:
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    touch ${prefix}.transcripts.gtf
+    touch ${prefix}.gene.abundance.txt
+    touch ${prefix}.coverage.gtf
+    touch ${prefix}.ballgown
```

Comment:
This file is missing in the files section of test.yml.

Reply:
Added